paddle/fluid/operators/detail/variable_response.h

```diff
@@ -61,7 +63,7 @@ class VariableResponse {
   // other: number of error field.
   int Parse(const ::grpc::ByteBuffer& byte_buffer);
 
-  const framework::Scope& GetLocalScope() const { return *local_scope_; }
+  framework::Scope& GetLocalScope() const { return *local_scope_; }
```

Comment:
Consider GetMutableLocalScope that returns a pointer and avoid removing the const?

Reply:
done